src/WebEncoders/src/Microsoft.Extensions.WebEncoders.csproj

```diff
@@ -7,6 +7,8 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore</PackageTags>
+    <IsPackable>true</IsPackable>
```

Comment:
I'd like to see if we can avoid having to define these two properties in every .csproj file. We already have convention-based settings in Directory.Build.props, and we also have the data to infer these values in most cases.